components/shared/net/indexeddb_thread.rs

```diff
@@ -192,11 +192,20 @@ pub enum SyncOperation {
     Exit(IpcSender<()>),
 }
 
+/// The set of all kinds of results that can be returned from async operations.
+#[derive(Debug, Serialize, Deserialize)]
+pub enum IdbResult {
+    /// A key.
+    Key(IndexedDBKeyType),
```

Comment:
This doc comment isn't great ^^.

I'd either remove it or make it more descriptive (what is the key for?)